pkg/util/docker/network.go

```diff
@@ -120,11 +120,19 @@ func findDockerNetworks(containerID string, pid int, container types.Container)
 		if fields[1] == "00000000" {
 			continue
 		}
-		dest, _ := strconv.ParseInt(fields[1], 16, 32)
-		mask, _ := strconv.ParseInt(fields[7], 16, 32)
-		for net, gw := range dockerGateways {
+		dest, err := strconv.ParseUint(fields[1], 16, 32)
```

Comment:
Shouldn't this be 64 bits instead of 32 now?

Reply:
IPv4 should always fit in 32 bits, and `/proc/net/route` is IPv4 only, being `/proc/net/ipv6_route` the source for ipv6 routing tables.